app/Http/Controllers/Backend/AdminController.php

```diff
@@ -78,4 +78,28 @@ public function store(AdminRequest $request)
 
         return redirect()->route('admin.admins.index');
     }
+
+    /**
+     * Remove the specified resource from storage.
+     *
+     * @param int $id id user
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function destroy($id)
```

Comment:
[rank C] please check this function, what's happen if admin delete all admin users.